lib/messages.js

```diff
@@ -941,5 +973,7 @@ Buffers.prototype.skip = function(i) {
 
 [Inventory, GetData, NotFound].forEach(function(clazz) {
   clazz.forBlock = creatorForItem(clazz, Inventory.TYPE.BLOCK);
+  clazz.forFilteredBlock = creatorForItem(clazz, Inventory.TYPE.FILTERED_BLOCK);
+  clazz.forMerkleBlock = creatorForItem(clazz, Inventory.TYPE.FILTERED_BLOCK);
```

Comment:
is the type the same on purpose or a typo? so, `forMerkleBlock` and `forMerkleBlock` are aliases?
in that case I would do:
```
clazz.forMerkleBlock = clazz.forFilteredBlock;
```